README.md

```diff
@@ -1,3 +1,7 @@
+## Live Link
+
+[Heroku](https://jaak-calculator.herokuapp.com/)
+
```

Comment:
**Kindly update your `README` file and remove unnecessary content**.
Checkout out the below link for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/